src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -195,6 +195,14 @@ func importConfig(
 
 	fmt.Printf("Successfully updated configuration. New configuration version is %d.\n", result.ConfigVersion)
 
+	status, err := client.ClusterConfigStatus(ctx, true)
+	out.MaybeDie(err, "error fetching status: %v", err)
```

Comment:
It’s better print more informative message here? Something like ‘Configuration is updated successfully but the subsequent config status check has failed. Please check the cluster status.' 
@twmb @r-vasquez

Reply:
I guess it's redundant.

Reply:
Might be redundant but it could be weird to have a success message of the original intention of the command (set/import)... and then the command fails with something that's unknown to the user (a status check after the command execution).

Maybe something like

`out.MaybeDie(err, "unable to check if the cluster needs to be restarted :%v\nCheck the status with 'rpk cluster config status':, err)`

wdyt?

Reply:
Thanks for the suggestion. That makes sense.